static/js/composebox_typeahead.js

```diff
@@ -320,6 +320,11 @@ exports.compose_content_begins_typeahead = function (query) {
     }
 
     if (this.options.completions.mention && current_token[0] === '@') {
+        // Don't autocomplete if there is a space following an '@'
+        if (current_token[1] === " ") {
+            return false;
+        }
+
```

Comment:
I think we might want this further down in the function; if you compare with the existing code for `#`, I think that the other will do better for syntax like `@**<space>`